Measurements/Tests.qs

```diff
@@ -349,6 +349,17 @@ namespace Quantum.Kata.Measurements {
 
         CheckSuperpositionBitstringsOneMeasurement(5, [30,14,10,6],     // [11110,01110,01010,00110]
                                                       [1,17,21,25]);    // [00001,10001,10101,11001]
+        CheckSuperpositionBitstringsMeasurement(2, [1,2],       // [01,10]
+                                                   [1]); // [01]
+
+        CheckSuperpositionBitstringsMeasurement(3, [7,5],       // [111,101]
+                                                   [2]); // [010]
+
+        CheckSuperpositionBitstringsMeasurement(4, [13,11,7,3], // [1101,1011,0111,0011]
```

Comment:
I think this test violates the constraint on the input for the task?

    // You are guaranteed that there exists an index of a qubit Q for which 
    //  - all the bit strings in the first array have the same value in this position (all bits1[j][Q] are the same),
    //  - all the bit strings in the second array have the same value in this position (all bits2[j][Q] are the same),
    //  - these values are different for the first and the second arrays.

Reply:
done